frontend/src/components/SignIn/SignIn.js

```diff
@@ -91,6 +103,7 @@ class SignIn extends Component {
           </div>
 
         </form>
+        <p>{err}</p>
```

Comment:
Should this `p` element have the `errorMessage` class to ensure it gets styled correctly?